pages/downloads.md

```diff
@@ -85,16 +85,17 @@ available in a future update to this site.
 {% for srcbin in srcbinArray %}
 {% assign lnkFile = 'accumulo-' | append: linkVers | append: '-' | append: srcbin | append: '.tar.gz' %}
 {% assign lnkSuffix = '/accumulo/' | append: linkVers | append: '/' | append: lnkFile %}
-<div class="row btn-group" style="margin-left: 20px; margin-bottom: 5px; display: block">
-  <div class="col btn-group">
+<div class="d-flex flex-wrap justify-content-start align-items-start" style="margin-left: 20px; margin-bottom: 5px;">
+  <div class="btn-group">
     <a {{btnDownloadStyle}} href="{{closerLink}}{{lnkSuffix}}" link-suffix="{{lnkSuffix}}">{{lnkFile}}{{glyphSave}}</a>
-  </div><div class="col btn-group">
+  </div>
+  <div class="btn-group">
```

Comment:
I'm not familiar with any of these classes and how they affect wrapping, but I'd be careful about introducing whitespace between the two divs. The old one had them directly adjacent, with no whitespace in-between, so the padding and wrapping worked correctly. Changing it is probably fine, so long as you've tested it. If it gets quirky, we can always revisit it and fix it later as a follow-on.